drivers/soundwire/bus.c

```diff
@@ -880,20 +880,26 @@ static int sdw_handle_slave_alerts(struct sdw_slave *slave)
 
 	sdw_modify_slave_status(slave, SDW_SLAVE_ALERT);
 
+	ret = pm_runtime_get_sync(&slave->dev);
+	if (ret < 0) {
```

Comment:
can you add the usual && ret !=  -EACCESS 

this is needed in case the device doesn't support pm_runtime